README.md

```diff
@@ -1,43 +1,39 @@
-# Hierarchical Infinite Relational Model
+# GenDB
 
 [![Actions Status](https://github.com/probcomp/hierarchical-irm/workflows/Python%20package/badge.svg)](https://github.com/probcomp/hierarchical-irm/actions)
 [![pypi](https://img.shields.io/pypi/v/hirm.svg)](https://pypi.org/project/hirm/)
 
-This repository contains implementations of the Hierarchical Infinite
-Relational Model (HIRM), a Bayesian method for automatic structure discovery in
-relational data. The method is described in:
+This repository contains an implementation of GenDB, a Bayesian method for
+automatic structure discovery in tabular and relational data.
 
-Hierarchical Infinite Relational Model. Saad, Feras A. and Mansinghka, Vikash K.
-In: Proc. 37th UAI, 2021.
-<https://proceedings.mlr.press/v161/saad21a.html>
+A paper describing this method is in preparation.  Briefly, this codebase
+builds on the [Hierarchical Infinite Relations
+Model](https://proceedings.mlr.press/v161/saad21a.html) implemented
+in <https://github.com/probsys/hierarchical-irm> and
+adds:
 
-## Installation (Python)
+1. Many more distribution types, including a normal distribution for floating
+point data, categorical and Skellam distributions for integer data, and a
+bigram distribution for string data.
 
-This software is tested on Ubuntu 18.04+ and requires a Python 3.6+
-environment. The library can be installed from the PyPI repository using
+2. Hyperparameter inference.
 
-    $ python -m pip install hirm
+3. Support for distributions like Skellam without conjugate priors.
 
-The library will be available as a module named `hirm`.
+4. Support for "noisy relations", which add noise to the output of another
+relation.
 
-The test suite can be invoked via
+5. A library of emission classes for noisy relations, including a gaussian
+emission for adding gaussian noise to real data, a bitflip emission for
+adding noise to boolean data, and a bigram emission for adding insertions,
+substitutions and deletions to string data.
 
-    $ python -m pytest --pyargs hirm
+In addition, we use these capabilities to build an entirely new interface
+to the HIRM model:  the pclean binary (named after the
+[PClean paper](https://proceedings.mlr.press/v130/lew21a/lew21a.pdf) on which
+it is heavily based) reads in tabular data and a schema describing the entities
+that generated it, and converts the data and schema into their HIRM equivalents.
 
-## Running Python examples
```

Comment:
FWIW I am thinking of removing a lot of the python subdirectory since it's not relevant (and can be recovered in probsys/hierarchical-irm). Happy for you to also do it (or neither of us do it at all), in case there are stuff you need for viz.

Reply:
Maybe wait one day?  I'll look at it tomorrow and see if there is anything I need for the visualization work.